packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -383,7 +382,7 @@ export function makeRumPublicApi(
         deflateWorker
       )
 
-      strategy = createPostStartStrategy(initConfiguration, startRumResult)
+      strategy = createPostStartStrategy(strategy.initConfiguration!, startRumResult)
```

Comment:
Suggestion: keep passing `initConfiguration` in `doStartRum` so you don't need this non-null assertions

Reply:
In fact I think of passing the all preStartStrategy as param cf commit https://github.com/DataDog/browser-sdk/pull/2795/commits/0c3d32fca91e0648cc5545945b1f84d2053b29a6
Wdyt?